scripts/dts/gen_defines.py

```diff
@@ -858,6 +858,16 @@ def write_global_macros(edt):
     # Global or tree-wide information, such as number of instances
     # with status "okay" for each compatible, is printed here.
 
+
+    out_comment("Macros for iterating over all nodes and enabled nodes")
+    out_define(
+        "DT_FOREACH_HELPER(fn)",
```

Comment:
I'm not sure how much readability difference this makes, but `out_dt_define` would work here.